standard/template/sections/clause_9_zarr_encoding_overviews.adoc

```diff
@@ -0,0 +1,101 @@
+
+=== Encoding of Multiscale Overviews in Zarr
+
+This clause specifies how multiscale tiling (also known as overviews or pyramids) is encoded in Zarr-based datasets conforming to the unified data model. The encoding supports both Zarr Version 2 and Version 3 and is aligned with the OGC Two Dimensional Tile Matrix Set Standard.
+
+Multiscale datasets are composed of a set of Zarr groups representing multiple zoom levels. Each level stores coarser-resolution resampled versions of the original data variables.
+
+==== Hierarchical Layout
+
+Each zoom level SHALL be represented as a Zarr group, identified by the Tile Matrix identifier (e.g., `"0"`, `"1"`, `"2"`). These groups SHALL be organised hierarchically under a common multiscale root group. Each zoom-level group SHALL contain the complete set of variables (Zarr arrays) corresponding to that resolution.
+
+[cols="1,2,2"]
+|===
+|Structure |Zarr v2 |Zarr v3
+
+|Zoom level groups | Subdirectories with `.zgroup` and `.zattrs` | Subdirectories with `zarr.json`, `node_type: group`
+
+|Variables at each level | Zarr arrays (`.zarray`, `.zattrs`) in each group | Zarr arrays (`zarr.json`, `node_type: array`) in each group
+
+|Global metadata | `multiscales` defined in parent `.zattrs` | `multiscales` defined in parent group `zarr.json` under `attributes`
+|===
```

Comment:
Unnecessary to refer explicitly to Zarr format details here. Let's just talk about Groups and Arrays, which is universal to both format versions.